gradle/dependencies.gradle

```diff
@@ -11,8 +11,7 @@ ext {
       mapboxSdkServices         : '5.3.0',
       mapboxEvents              : '6.0.0',
       mapboxCore                : '3.0.0',
-      mapboxNavigator           : '16.0.0',
-      mapboxCommonSdk           : '3.1.0', // Workaround. Remove when upgrading Nav Native.
+      mapboxNavigator           : '18.0.1',
```

Comment:
```suggestion
      mapboxNavigator           : '18.0.2',
```
![giphy (1)](https://user-images.githubusercontent.com/16925074/89403701-0407dd80-d719-11ea-8639-2ca180486283.gif)

Reply:
From `18.0.0` to `18.0.1` 😂

Reply:
done